docs/docs/topics/schema.mdx

```diff
@@ -29,10 +29,20 @@ To help with the development process of a schema definition file, you can levera
 
 The schema is composed of 4 primary types of objects: `Nodes`- that are themselves composed of `Attributes` and `Relationships` and finally `Generics`.
 
-- A `Node` in Infrahub represents a `Model`. Nodes are instances of a specific object within your infrastructure. Nodes have attributes that define specific values, such as text or numbers, and relationships that link them to other nodes.
-- An `Attribute` represents a direct value associated with a `Node` like a `Text`, a `Number` etc ...
-- A `Relationship` represents a link between 2 `Node`, a `Relationship` can be of cardinality `one` or `many`.
-- A `Generic` can be used to share attributes and relationships between different types of `Node`s. They can connect multiple types of nodes to the same relationship or define attributes and relationships on a specific list of nodes. Generics are similar to class inheritance in programming languages like Java or Python.
+* A `Node` in Infrahub represents a `Model`. Nodes are instances of a specific object within your infrastructure. Nodes have attributes that define specific values, such as text or numbers, and relationships that link them to other nodes.
+* An `Attribute` represents a direct value associated with a `Node` like a `Text`, a `Number` etc ...
+* A `Relationship` represents a link between 2 `Node`, a `Relationship` can be of cardinality `one` or `many`.
+* A `Generic` can be used to share attributes and relationships between different types of `Node`s. They can connect multiple types of nodes to the same relationship or define attributes and relationships on a specific list of nodes. Generics are similar to class inheritance in programming languages like Java or Python.
+
+:::info Reserved Namespaces & Attributes Names
+
+Some namespace like `Core`, `Infrahub` and `Builtin` are reserved and can't be used by users to defined new Nodes and Generics.
+The complete list is present in this [file](https://github.com/opsmill/infrahub/blob/stable/backend/infrahub/core/constants/__init__.py#L292)
```

Comment:
Maybe import this snippet in to the docs, so it's viewable? There's a way to import and display snippets in docusaurus.